packages/api/portfolioDrafts/application/createApplicationStep.test.ts

```diff
@@ -348,28 +348,29 @@ describe("Business rules validation tests", () => {
     expect(responseBody.details[0].message).toEqual("must NOT have less than 1 item");
     expect(result?.body).toMatch(/"\/body\/applications\/0\/environments"/);
   });
-  it("should return a validation error when admin roles are not acceptable", async () => {
```

Comment:
Is this a good opportunity to use `it.ignore()` ?

Reply:
Didn't know that was a thing, i'll try it

Reply:
Didn't see `it.ignore()` but used `it.skip()` instead.

Reply:
That is probably what I meant.  😄